chromium_src/chrome/browser/themes/theme_service_factory.cc

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "build/build_config.h"  // For OS_LINUX
+
```

Comment:
we need to include theme_service.h before we do anything else so we don't accidentally modify it

Reply:
done.